assets/styles/_responsive.scss

```diff
@@ -218,16 +237,22 @@
   }
 
   .header-carousel {
-    height: 97vh !important;
+    height: 100vh !important;
 
     .overlay {
       height: 27rem !important;
     }
 
     .slider-overlay-container {
-      margin-top: -97vh;
+      margin-top: -100vh;
     }
   }
+
+  .header-carousel .overlay {
```

Comment:
let's refrain from changing heights and widths from now. let's focus on the fonts and typography.